repos/newtonsoft-json.proj

```diff
@@ -24,7 +28,7 @@
   </PropertyGroup>
 
   <Target Name="Restore" BeforeTargets="Build" DependsOnTargets="UpdateNuGetConfig">
-    <Exec Command="$(DotnetToolCommand) restore $(NewtonsoftJsonProjectPath) $(DotnetToolCommandArguments) /v:$(LogVerbosity) $(RedirectRepoOutputToLog)"
+    <Exec Command="$(DotnetToolCommand) restore $(NewtonsoftJsonProjectPath) $(DotnetToolCommandArguments) /v:$(LogVerbosity) /bl $(RedirectRepoOutputToLog)"
```

Comment:
Suggest adding binlogging to the `BuildCommand` and `BuildPackagesCommand` as well - you'll have to name them differently (e.g. `/bl:restore.binlog`).